docs/fidesops/docs/postman/Fidesops.postman_collection.json

```diff
@@ -2535,6 +2535,229 @@
 						}
 					]
 				},
+				{
+					"name": "Segment",
```

Comment:
This is a helpful addition, do you think we should add a new collection for each integration we add?

Reply:
I mean I don't want to go overboard, but this was helpful for myself when I was testing.  I also duplicated the collection and edited for Stripe when I was testing your code. I think it's useful in the beginning.